packages/DelphiModbus12Athens.dpk

```diff
@@ -0,0 +1,46 @@
+package DelphiModbus12Athens;
+
+{$R *.res}
+{$IFDEF IMPLICITBUILDING This IFDEF should not be used by users}
+{$ALIGN 8}
+{$ASSERTIONS ON}
+{$BOOLEVAL OFF}
+{$DEBUGINFO OFF}
+{$EXTENDEDSYNTAX ON}
+{$IMPORTEDDATA ON}
+{$IOCHECKS ON}
+{$LOCALSYMBOLS ON}
+{$LONGSTRINGS ON}
+{$OPENSTRINGS ON}
+{$OPTIMIZATION OFF}
+{$OVERFLOWCHECKS OFF}
+{$RANGECHECKS OFF}
+{$REFERENCEINFO ON}
+{$SAFEDIVIDE OFF}
+{$STACKFRAMES ON}
+{$TYPEDADDRESS OFF}
+{$VARSTRINGCHECKS ON}
+{$WRITEABLECONST OFF}
+{$MINENUMSIZE 1}
+{$IMAGEBASE $400000}
+{$DEFINE DEBUG}
+{$ENDIF IMPLICITBUILDING}
+{$DESCRIPTION 'Delphi Modbus components runtime package'}
+{$RUNONLY}
+{$IMPLICITBUILD OFF}
+
+rrequires
+  rtl,
+  IndyCore,
+  IndyProtocols,
+  IndySystem;
+
+ocontains
```

Comment:
Invalid syntax 'oncontains'

Reply:
and at end there is "n." instead if "end.", sorry funny IDE bug hit me again.
It should be fixed!